force-app/main/default/classes/RecurrenceRule.cls

```diff
@@ -100,11 +100,85 @@ public with sharing class RecurrenceRule {
 
     public String build() {
```

Comment:
Can we call this method buildRRuleString or something else?

Reply:
`buildRRule`